types/index.d.ts

```diff
@@ -27,7 +27,7 @@ export interface ColorSupport {
 
 export interface Chalk {
 	(...text: string[]): string;
```

Comment:
This should be changed too.

```suggestion
	(...text: unknown[]): string;
```

We coerce this argument to string: https://github.com/chalk/chalk/blob/076f0c9eb6d6575db745f2753832918eaefe627d/index.js#L167